src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -160,6 +160,19 @@ public function thereIsCustomerThatPlacedOrder(CustomerInterface $customer, $ord
     }
 
     /**
+     * @Given the customer :customer have already placed an order :orderNumber
```

Comment:
Above, we already have method for step `@Given a customer :customer placed an order :orderNumber` we can either reuse this step in scenarios or assign this step to that method.